core/client/src/main/java/uk/gov/dwp/queue/triage/core/client/SearchFailedMessageClient.java

```diff
@@ -11,5 +15,9 @@
 
     @GET
     @Path("/{failedMessageId}")
-    FailedMessageResponse getFailedMessage(@PathParam("failedMessageId") FailedMessageId failedMessgeId);
+    FailedMessageResponse getFailedMessage(@PathParam("failedMessageId") FailedMessageId failedMessageId);
+
+    @GET
+    @Path("/{broker}/count")
+    long getNumberOfFailedMessages(@PathParam("broker") String broker);
```

Comment:
This endpoint will more than likely be removed (or replaced) when search functionality is introduced.  I thought it was better to expose an endpoint for the component-tests rather than reaching into the database directly.